rspace/src/main/scala/coop/rchain/rspace/LMDBStore.scala

```diff
@@ -139,24 +159,30 @@ class LMDBStore[F[_], C, P, A, K] private[rspace] (
       txn: Transaction,
       channels: Seq[C],
       continuation: WaitingContinuation[P, K]
-  ): Unit =
+  ): Unit = {
+    val cloned       = LMDBStore.roundTrip(continuation)
```

Comment:
Why is this needed?

Reply:
because there is no guarantee that the continuation wont have internal state